queries/readline/highlights.scm

```diff
@@ -0,0 +1,69 @@
+(comment) @comment @spell
+
+"set" @keyword
+
+[
+  "$if"
+  "$else"
+  "$endif"
+] @keyword.conditional
+
+"$include" @keyword.import
+
+[
+  "="
+  "=="
+  "!="
+  ">="
+  "<="
+  ">"
+  "<"
+] @operator
+
+[
+  ":"
+  "-"
+] @punctuation.delimiter
+
+[
+  (application_name)
+  (keyname)
+  (keyseq)
+  (macro)
+  (string_value)
+  (term_name)
+] @string
+
+(file_path) @string.special.path
+
+(escape_sequence) @string.escape
+
+(symbolic_character_name) @constant.builtin
+
+(key_literal) @character.special
+
+(function_name) @function
+
+[
+  (bell_variable)
+  (bool_variable)
+  (edit_mode_variable)
+  (keymap_variable)
+  (number_variable)
+  (string_variable)
+] @variable.builtin
+
+[
+  (bell_value)
+  (edit_mode_value)
+  (keymap_value)
+  "version"
+  "term"
+  "mode"
+] @constant.builtin
+
+(number_value) @number
+
+(version_number) @number.float
```

Comment:
I am not so sure with this change: I think it would be good if the version number is always prefixed with a `v`, but it looks like it is always just version number alone. Also it is used with comparison operators like `>` or `<=` which I think makes more sense in a numerical context. But I will commit if you still think special string is better.

Reply:
Good point

Reply:
Upon further inspection I think we would not have to deal with things like `1.0.0` since readline does not use standard semantic versioning numbers, as seen in [their changelog](https://tiswww.case.edu/php/chet/readline/CHANGES). Also their syntax description backs up that only one decimal point may be used:

> The version number supplied on the right side of the operator consists of a major version number, an optional decimal point, and an optional minor version (e.g., ‘7.1’). If the minor version is omitted, it is assumed to be ‘0’.

However, right now the parser allows arbitrary amounts of decimal points so maybe it is best to keep it as `@string.special` now and then if we feel the need we can change back to `@number.float` after the parser is updated